### PR TITLE
data: AyushRay06

### PR DESCRIPTION
**Pull request by AyushRay06**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You'll are doing a great job. I would like my LinkFree profile to be updated.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8465"></a>

### CODE REVIEW

File: data/AyushRay06.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	"name": "Ayush Ray",
+	"bio": "DevOps| webdev| cloud| GDSC lead",
+	"tags": [
+		"Open Source",
+		"MERN",
+		"Kubernaties",
+		"AWS",
+		"Docker"
+	],
+	"Socials": [{
```
Comment (sital002) [inline data/AyushRay06.json:11]: Socials shouldn't be capitalized. It should be socials

**Review by sital002 — CHANGES_REQUESTED** [review]

File: data/AyushRay06.json (hunk 1 above)
Comment (sital002) [inline data/AyushRay06.json:11]:
```suggestion
	"socials": [{
```

Comment (sital002) [reply data/AyushRay06.json:11]: "Socials" shouldn't be capitalized. It should be "socials"

**Review by sital002 — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**